magento-research/pwa-studio
# [Documentation] Add markdown linting

## Description

Creates a set of standards for the markdown used in documentation.
Adds the ability to lint a single file or directory.

## Related Issue

Closes #1127 

## Verification Steps

1. Navigate to the `pwa-devdocs` directory
2. Run `npm run tests`
3. Verify that warnings are thrown on unfixed topics.

## How Have YOU Tested this?

See **Verification Steps**

## Screenshots / Screen Captures (if appropriate):

n/a

## Proposed Labels for Change Type/Package


## Checklist:
